docs/03-module-block-3/02-learning-goals.adoc

```diff
@@ -2,39 +2,62 @@
 
 // tag::DE[]
 [[LZ-3-1]]
+==== LZ 3-1: Kombinatormodelle
 
-- Was sollen die Teilnehmer können?
-   * DDD mit Hilfe funktionaler Bausteine umsetzen
-   * rudimentäre Kombinatormodelle formulieren
-   * kleine domänenspezifische Sprachen entwerfen
+* die Struktur von Kombinatormodellen mit Selbstbezüge verstehen
```

Comment:
Der Begriff "Selbstbezug" taucht hier auf und auch noch anderswo. Ich bin mir nicht sicher, ob die intendierte Bedeutung auch allen klar wird. Gemeint sind (?) hier doch keine zyklischen Objekte, wie bspw. Graphen, die in der funktionalen Modellierung eine besondere Herausforderung darstellen.